content/developer/tutorials/data_modules.rst

```diff
@@ -0,0 +1,906 @@
+
+Computed Fields
+---------------
+
+Computed fields are an core concept in Odoo and are used to define fields that are computed
+based on other fields. This is useful for fields that are derived from other fields, like a
+sum of sub-records (adding up the price of all the items in a sale order).
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`reference/fields/compute`.
+
+Data modules can define computed fields of any type, but are quite limited compared to Python
+modules. Indeed, since data modules are meant to be deployed on systems that do not allow arbitrary
+code to run, the Python code that is allowed is very limited.
+
+.. note::
+
+    All Python code written for data modules is executed in a sandboxed environment that limits
+    the operations that can be performed. For example, you cannot import libraries, you cannot
+    access any OS files, and you cannot even print to the console. Some utilities are provided,
+    but this varies with the type of sandboxed environment that is used.
+
+    In the case of compute methods, the sandbox is very limited and only provide the bare minimum
+    of utilities to allow the execution of the code. In addition to the Python builtins, you also
+    have access to the `datetime`, `dateutil` and `time` modules (e.g. to help with date
+    calculations).
+
+    Note also that "dot assignation" is disabled in the sandbox, so you cannot write
+    ``property.x_total_area = 1`` in the compute method. You have to use dictionary access:
+    ``property['x_total_area'] = 1``. Dot notation for field *access* works normally:
+    ``property.x_garden_area`` will return the value of the ``garden_area`` field.
+
+
+We previously defined two "area" fields on our ``x_estate.property`` model: ``living_area``
+and ``garden_area``. To define a computed field on the model that returns the sum of the two
+areas, we can add the following code to our data module:
+
+.. code-block:: xml
+
+    <odoo>
+        <!-- ...model definition from before... -->
+        <record model="ir.model.fields" id="field_real_estate_property_total_area">
+            <field name="model_id" ref="estate.model_real_estate_property" />
+            <field name="name">x_total_area</field>
+            <field name="field_description">Total Area</field>
+            <field name="ttype">float</field>
+            <field name="depends">x_living_area,x_garden_area</field>
+            <field name="compute"><![CDATA[
+    for property in self:
+        property['x_total_area'] = property.x_living_area + property.x_garden_area
+            ]]>
+        </field>
+        </record>
+    </odoo>
+
+The ``depends`` attribute is used to define the fields that the computed field depends on and
+the ``compute`` attribute is used to define the code that is executed to compute the field.
+
+Computed fields are not stored in the database by default, however they can be stored by setting
+the ``store`` attribute to ``True``.
```

Comment:
indeed! interesting, did not know that. don't think it should be tackled in base; i prefer having a 'standard' default value like True rather than "True for non-compute but False for compute fields".

will adapt accordingly, nice catch!